### PR TITLE
Fix Yahoo Weather API calls

### PR DESCRIPTION
**Pull request by fourstix**
Yahoo recently changed their API so that the forecast.py and timetemp.py scripts no longer work.  This patch updates the yahoo url in the script to change http://weather.yahooapis.com to http://xml.weather.yahoo.com to fix the error.

### CODE REVIEW

File: timetemp.py [hunk 1]
```diff
@@ -27,7 +27,8 @@
 
 # Fetch weather data from Yahoo!, parse resulting XML
 dom = parseString(urllib.urlopen(
-        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())
+#        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())
+        'http://weather.yahooapis.com/forecastrss?w=' + WOEID).read())
```
Comment (neildaniels) [inline timetemp.py]: This line is identical to the one it "replaces".

Comment (fourstix) [inline timetemp.py]: My bad.  I have updated the files with the correct code changes in my branch and re-issued the pull requests.  They should update the code to use the YQL format for the Yahoo weather API.

**Comment by ladyada** [comment]
hiya there's some conflicts, dya want to submit a clean PR?

**Comment by fourstix** [comment]
Hi, We can just cancel it.  It's really old.
 Yahoo has changed things since then, so this fix no longer works. 
I'll just close this request.
Have a great day.

**Pull request closed without merging**